src/components/NewsTeaserItem/__tests__/NewsTeaserItem.spec.tsx

```diff
@@ -0,0 +1,55 @@
+import { render, fireEvent, getByTestId } from "@testing-library/vue";
+import NewsTeaserItem from "./../";
+
+describe("components/NewsTeaserItem", () => {
+  it("calls handleClick callback on call", async () => {
+    const title = "title test";
```

Comment:
Please keep it DRY and define these constants outside of your tests since you use the same test values in each test.